source/develop/release-management/features/storage/incremental-backup.md

```diff
@@ -496,6 +519,14 @@ missing in the image, or unknown bitmap exist, all the checkpoints on
 the disk and engine database must be deleted, and the current backup
 must be a full backup.
 
+### Setting the disks backup_mode before starting an incremental backup
+
+Before starting an incremental backup, the system will validate that all
+the disks that participate in the incremental backup already backed-up in
+in the given checkpoint ID. For disks which are part of the given checkpoint ID
+the backup mode will be `incremental`, if the disks are not part of the given
+checkpoint, the backup mode will be `full`.
```

Comment:
This should also work for the case when we cannot syncronize checkpoints with libvirt. I think currently we fail the backup. Instead, all disks should be marked as backup_mode="full". This eliminates the error for this case.

Reply:
added